### ISSUE TITLE
"Something went wrong" (Nextcloud)

### ISSUE THREAD

**Issue by gerega** [msoultanidis/quillnote#128]
Getting a not helpful message while trying to login with the correct credentials to Nextcloud. Any help appreciated! I am logging in with the app-token.

**Comment by KonstantMan** [comment]
+1
And yes i have i enabled https

**Comment by KonstantMan** [comment]
@gerega  In order to work you have to enable the notes app in nextcloud but even then it syncs every note as a txt file for me that's a deal breaker unfortunately i don't want my folders full of txt files

I once again hope for a more polished solution with caldav or i will just use it only offline

**Comment by gerega** [comment]
@KonstantMan Yes, enabling the Notes app will make the synchronization work. Thank you a lot!
